releases/8.0.0.md

```diff
@@ -69,9 +72,9 @@ JavaScript:
 
 1. In settings.py, add the following key to `DATABASES` to [improve indexing performance](https://github.com/archesproject/arches/issues/11382):
     ```
-        "OPTIONS": {
-            "options": "-c cursor_tuple_fraction=1",
-        },
+    "OPTIONS": {
+        "options": "-c cursor_tuple_fraction=1",
+    },
```

Comment:
I had this intended like that for easier copy-pasting, since it's two levels deep. Maybe it's better to keep the indenting and clarify in the verbiage that this is all underneath "default:".

Reply:
oh no fair enough, I just ran the doc through a formatter and that popped out. Happy to revert